src/webui/webapplication.cpp

```diff
@@ -252,6 +252,9 @@ void WebApplication::doProcessRequest()
     const QString action = match.captured(u"action"_qs);
     const QString scope = match.captured(u"scope"_qs);
 
+    if (!session() && !isPublicAPI(scope, action))
+        throw ForbiddenHTTPError();
+    
```

Comment:
Seems file health checker complaints about this line.

Reply:
@RequiDev "trailing whitespaces"

![spacing](https://user-images.githubusercontent.com/42386382/163030521-ef894bfc-922a-4ceb-8a29-027a0373491e.png)

Reply:
Fixed it.